Readme.md

```diff
@@ -2,7 +2,7 @@
 
 ## PHP Client
 
-### Pre requisits
+### Pre requisites
 1. a working php version > 5.3
 2. composer should be installed, see [composer](http://www.getcomposer.org) for details how to install
```

Comment:
Videos with South Africans accent? I wouldn't recommend it, no-one would understand ;)